.clang-format

```diff
@@ -18,6 +18,9 @@ AllowShortLambdasOnASingleLine: None
 AllowShortLoopsOnASingleLine: 'false'
 AlwaysBreakAfterDefinitionReturnType: None
 AlwaysBreakAfterReturnType: None
+BinPackArguments: 'false'
+BinPackParameters: 'false'
+BreakAfterAttributes: Always
```

Comment:
This is a new setting introduced in ClangFormat 16.0. It ensures that ClangFormat does not move attributes to the same line as the rest of the definition. ie.
```c++
[[deprecated]]
void DoSomething();
```
instead of
```c++
[[deprecated]] void DoSomething();
```